yt/visualization/base_plot_types.py

```diff
@@ -341,7 +341,15 @@ def _set_axes(self) -> None:
         self.image.axes.set_facecolor(self.colorbar_handler.background_color)
 
         self.cax.tick_params(which="both", direction="in")
-        self.cb = self.figure.colorbar(self.image, self.cax)
+        from mpl_toolkits.axes_grid1.axes_grid import CbarAxesBase
```

Comment:
is this import statement known expensive (long startup time) or could it be moved to the top level of the module ?

Reply:
The result from `timeit` is about 2 seconds. Would it be considered as expensive?

Reply:
Very much so, yes. I consider anything longer than .1s to me expensive for an import statement.